apps/ws/src/socket/services/web-socket.worker.ts

```diff
@@ -41,14 +42,16 @@ export class WebSocketWorker extends WebSocketsWorkerService {
           'WS Service',
           function () {
             const transaction = nr.getTransaction();
+            const { data: jobData } = job;
```

Comment:
Good point, however, these two id's aren't connected. Adding specific types helps open our eyes to better understand the types that are in the field. However, our current solution isn't completely typed.

To clarify, the 'name' serves as the Job's name, but it's relatively redundant now. It was initially used in the 'bull' library to distribute jobs among workers. However, 'bullmq' has deprecated this to avoid confusion. [more info here](https://github.com/taskforcesh/bullmq/issues/297#issuecomment-715406744
).

The job ID you mentioned is an internal ID used by 'bullmq'. To assign it, in order to override it we need to pass it as 'option.jobId' in the third 'add' parameter [more about it here](https://docs.bullmq.io/guide/jobs/job-ids).